package.json

```diff
@@ -87,6 +87,7 @@
     "node-fetch": "^3.1.1",
     "node-stream-zip": "^1.15.0",
     "p-wait-for": "^4.1.0",
+    "path-key": "^4.0.0",
```

Comment:
The diff in `package-lock.json` seems pretty large. Are you sure that's all due to the addition of `path-key`?

Reply:
Ah that might have been the forth an back between different dependencies. Will fix.